.github/workflows/pr-auto-assign.yml

```
@@ -16,11 +16,21 @@ jobs:
           script: |
             const reviewers = ['mstoykov', 'imiric', 'codebien', 'olegbespalov', 'oleiade'];
             const reviewerCount = 2;
+            const crypto = require("node:crypto");
+
+            const getNRandom = (n, array) => {
+              let result = new Array();
+              for (;n > 0 && array.length > 0; n--) {
+                const chosen = array[crypto.randomInt(array.length)];
```

BTW, we don't need to use `node:crypto` here. The PRNG would work just as well, and probably have less overhead.

So this works too:
```suggestion
                const chosen = array[Math.floor(Math.random()*array.length)];
```

We could also use the Fisher-Yates algorithm described [here](https://javascript.info/array-methods#shuffle-an-array) (click on "solution"), which is likely even more efficient.

Not that efficiency matters at all for this purpose, and this is fine too. Just wanted to mention it in case someone else finds it interesting as well. :slightly_smiling_face: